master/docs/manual/credentialmanagement.rst

```diff
@@ -0,0 +1,100 @@
+Credential Management
```

Comment:
The file should be name cfg-credentials.rst and be acessible from the config.rst topic index